manifests/m/Martchus/syncthingtray/1.4.1/Martchus.syncthingtray.installer.yaml

```diff
@@ -1,16 +1,15 @@
-# Created with Komac v1.6.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: Martchus.syncthingtray
 PackageVersion: 1.4.1
 InstallerType: zip
 NestedInstallerType: portable
 NestedInstallerFiles:
-- RelativeFilePath: syncthingctl-qt5-1.4.1-x86_64-w64-mingw32.exe
+- RelativeFilePath: syncthingtray-qt5-1.4.1-x86_64-w64-mingw32.exe

```

Comment:
It would make sense to use the Qt 6 based version. I only provide the Qt 5 based version for compatibility with Windows versions older than 10. However, for Windows 10 and newer the Qt 6 based version has some beneficial improvements like better High-DPI and darkmode support.